lib/spoom/sorbet/sigils.rb

```diff
@@ -56,14 +56,14 @@ def self.update_sigil(content, new_strictness)
       sig { params(path: T.any(String, Pathname)).returns(T.nilable(String)) }
       def self.file_strictness(path)
         return nil unless File.exist?(path)
-        content = File.read(path)
+        content = File.read(path, encoding: 'UTF-8')
```

Comment:
Will this solve the problem? We're still reading an ISO file as UTF8?

Furthermore, would that mean we'll save the file as UTF after the bump? Silently re-encoding users files might not be the best approach here.